src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -136,6 +142,32 @@
     /** mx4jport */
     MX4JPORT ("mx4jport"),
 
+    /**
+     * Gossip quarantine delay is used while evaluating membership changes and should only be changed with extreme care.
+     */
+    GOSSIPER_QUARANTINE_DELAY("cassandra.gossip_quarantine_delay_ms"),
```

Comment:
Do we set this at more than one places? Can't we add here the value you set for override (I didn't look in detail in that property but fast grep, I didn't see we set it more than once?)

Reply:
we set in a single place, but the default value is dependent on another config, so can't set a default; default is `RING_DELAY * 2`

Reply:
Yeah, makes sense